shared/version/api.go

```diff
@@ -401,6 +401,7 @@ var APIExtensions = []string{
 	"container_syscall_filtering_allow_deny_syntax",
 	"access_management",
 	"vm_disk_io_limits",
+	"instances_files_modify_permissions",
```

Comment:
please add api extension and associated doc change as a separate single commit, e.g. like 

https://github.com/canonical/lxd/commit/5d1ab06926d2cfa9d91324a832a3eca5544fd507

See https://github.com/canonical/lxd/blob/main/CONTRIBUTING.md#commit-structure

Reply:
Also for ease of backporting please make it the first commit in your PR so I can see what is coming up.